File: trunk/DotNetNuke/Sample/Presenters/DependencyUC2Presenter.vb
Imports DNN.MVP.Base.BasePresenter
Imports DNN.MVP.Sample.ModuleName1.IViews

Namespace Presenters
    Public Class DependencyUC2Presenter
        Inherits BasePresenter(Of IDependencyUC2View, Object)

        Public Sub New(view As IDependencyUC2View)
            MyBase.New(view)
        End Sub

        Public Overrides Sub ViewAction()
        End Sub

        Public Overrides Sub UpdateAction()
        End Sub

        Public Overrides Sub DeleteAction()
        End Sub

        Public Overrides Sub InsertAction()
        End Sub

        Public Overrides Sub InitializeAction()
        End Sub

        Public Overrides Sub SearchAction()
        End Sub

        Public Overrides Sub Validate()
        End Sub
    End Class
End Namespace

File: trunk/MVP.Sample/Backup/MVP.Sample.Web/Presenters/DependencyUC2Presenter.cs
using MVP.Base.BasePresenter;
using MVP.Sample.Web.IViews;

namespace MVP.Sample.Web.Presenters
{
    public class DependencyUC2Presenter: BasePresenter<IDependencyUC2View, object>
    {
        public DependencyUC2Presenter(IDependencyUC2View view) : base(view)
        {
        }

        #region Overrides of BasePresenter<IDependencyUC2View,object>

        protected override object Repository
        {
            get { return null; }
        }

        protected override void ViewAction()
        {
            //throw new NotImplementedException();
        }

        protected override void UpdateAction()
        {
            //throw new NotImplementedException();
        }

        protected override void DeleteAction()
        {
            //throw new NotImplementedException();
        }

        protected override void InsertAction()
        {
            //throw new NotImplementedException();
        }

        protected override void InitializeAction()
        {
            //throw new NotImplementedException();
        }

        protected override void SearchAction()
        {
            //throw new NotImplementedException();
        }

        public override void Validate()
        {
            //throw new NotImplementedException();
        }

        #endregion
    }
}